app/components/CreateAccountSlide/index.js

```diff
@@ -87,106 +92,142 @@ const SeedsContainer = styled.div`
 `
 
 type Props = {
-  seed: string,
-  currentSlide: number,
-  nextAccountSlide: Function,
-  importAddress: Function,
-  generateNewMnemonic: Function
+  importAddress: Function
 };
 
-const setupSeedColumns = (seed) => {
-  const seedWords = seed.split(" ");
-  const seeds = [];
-  let seedColums = [];
+class CreateAccountSlide extends Component<Props> {
+  props:Props;
+  state = {
+    seed: '',
+    currentSlide: 0
+  };
 
-  seedWords.forEach((item, index) => {
-    seedColums.push(item);
-    if (index%5 === 4) {
-      seeds.push(seedColums);
-      seedColums = [];
-    }    
-  });
+  componentDidMount() {
+    this.updateMnemonic();
+  }
 
-  return seeds;
-}
+  nextAccountSlide = (currentSlide) =>  this.setState({ currentSlide });
+  updateMnemonic = () =>  this.setState({ seed: generateNewMnemonic() });
 
-const ShowSeedPhrase =  ({ seed, nextAccountSlide, generateNewMnemonic }) => {
-  const seeds = setupSeedColumns(seed);
-  return (
-    <Fragment>      
-      <div className='description'>
-        Write down your seed phrase on a piece of paper and keep it in a safe place. You will need your seed phrase to recover your account.
-      </div>
-      {seeds.length && 
+  setupSeedColumns = (seed) => {
+    const seedWords = seed.split(" ");
+    const seeds = [];
+    let seedColums = [];
+
+    seedWords.forEach((item, index) => {
+      seedColums.push(item);
+      if (index%5 === 4) {
+        seeds.push(seedColums);
+        seedColums = [];
+      }
+    });
+
+    return seeds;
+  };
+
+  showSeedPhrase =  () => {
+    const seeds = this.setupSeedColumns(this.state.seed);
+    return (
+      <Fragment>
+        <div className='description'>
+          Write down your seed phrase on a piece of paper and keep it in a safe place. You will need your seed phrase to recover your account.
+        </div>
+        {seeds.length &&
         <SeedsContainer>
           { seeds.map((items, index) => {
-              return (
-                <div className="seedColumn" key={index}>
-                  {items.map((item, index1) => (<div key={index1} className="seedItem"><div>{index*5 + index1 + 1}</div>{item}</div>))}
-                </div>
-              )
-            })
+            return (
+              <div className="seedColumn" key={index}>
```

Comment:
Doesn't the seed have a hash or some other unique identifier you could use as the key rather than index? It's usually not a problem, but this has bitten me in the past